package.json

```diff
@@ -73,6 +75,7 @@
     "aws-sdk": "^2.894.0",
     "babel-loader": "^8.2.5",
     "babel-plugin-module-resolver": "^4.1.0",
+    "babel-plugin-transform-remove-strict-mode": "^0.0.2",
```

Comment:
This was probably added due to the test issues?? If so, we can remove this since we figured out what was causing those errors.

Reply:
100%, yes thank you